### DISCUSSION TITLE
Update the "About" description

### DISCUSSION THREAD

**Discussion by fralken** [versat/cntlm#76] [Ideas]
This project has evolved a lot since Evengard's repo, so I suggest to replace the last sentence of the "About" description with something like this:
_This version also supports: SSPI (on Windows, NTLM authentication only), Kerberos authentication, IPv6, proxy PAC files._

**Comment by versat** [comment]
Good idea. Sorry, I have somehow not seen this discussion until now 🙈
Done

**Reply by fralken** [reply]
No problem. Now we advertise all the new features. 🥇
